mne/report.py

```diff
@@ -977,6 +978,13 @@ def __init__(self, info_fname=None, subjects_dir=None,
             raise TypeError('raw_psd must be bool or dict, got %s'
                             % (type(raw_psd),))
         self.raw_psd = raw_psd
+
+        # Dictionary containing elements included in head
+        self.include = OrderedDict()
```

Comment:
So the `Report` class has historically not had documentation for its attributes. It might be a good time to do this while you're at it!

Reply:
I added documentation for attributes not occuring in Parameters. Or should I inlcude them as well?

Reply:
ummm ... include everything I think? How do other classes do it? Like `Epochs` for instance.

Reply:
Ok, in Epochs for example event_id is appearing in both `Parameters` and `Attributes`. The type-hint differs because the parameter accepts more. For the Report-Class, one would probably just copy the description from `Parameters`, right?

Reply:
yeah I think so!